grc/gui/Actions.py

```diff
@@ -514,8 +527,12 @@ def connect(action, handler=None):
     icon_name='printer',
     keypresses=["<Ctrl>p"],
 )
-PORT_CONTROLLER_DEC = actions.register("win.port_controller_dec")
-PORT_CONTROLLER_INC = actions.register("win.port_controller_inc")
+PORT_CONTROLLER_DEC = actions.register("win.port_controller_dec",
+    keypresses=["minus"],
```

Comment:
`PORT_CONTROLLER_DEC` and `PORT_CONTROLLER_INC` are missing the numpad keys.

The strings are `KP_Add` and `KP_Subtract`

Reply:
I appended the strings KP_Add and KP_Subtract in the list.